coreos-virt-install

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python3
+# Wrapper for virt-install designed to run in a container
+# environment, and using defaults targeted for CoreOS.
+# https://github.com/cgwalters/coreos-assembler/issues/7
+# Today, this requires a Kickstart (and an installer ISO path);
+# we might change things down the line to have defaults for
+# one or both.
+
+import os,sys,argparse,subprocess,io,time,re
+
+def fatal(msg):
+    print('error: ' + msg, file=sys.stderr)
```

Comment:
nit: In 3 `'words {} words'.format(var)` tends to be the preferred way to concat strings.